app/models/group.rb

```diff
@@ -108,8 +108,8 @@ def has_mod?(user)
     member(user).try(:mod?)
   end
 
-  def can_delete_story?(user)
-    has_admin?(user) || has_mod?(user)
+  def is_staff?(user)
+    !!(has_admin?(user) || has_mod?(user))
```

Comment:
Glad to see you renamed this, it was bothering me with how specific it was before! Also bang bang is the best operator :gun: :gun: